src/PluginStartupRegistration.cpp

```diff
@@ -0,0 +1,81 @@
+/**********************************************************************
```

Comment:
The sixth commit, which introduces this file, fails to compile.  It requires the overload of `RegisterPlugin` added at the next commit.

Reply:
Fixed